### ISSUE TITLE
Support lists in manifests

### ISSUE THREAD

**Issue by viglesiasce** [jenkinsci/google-kubernetes-engine-plugin#61]
It is common for Kubernetes users to have folders of manifests that they apply by passing multiple -f flags. For example `kubectl apply -f k8s/staging/ -f k8s/services/ `.

It is not currently possible to use multiple manifest paths per build step which means configuring multiple build steps in order to accomplish what is usually a single kubectl command.

**Comment by avadhut007** [comment]
yes we should be able to pass the list of config yamls in manifest option
